### ISSUE TITLE
Not working with AFNetworking 3.0

### ISSUE THREAD

**Issue by revolter** [luckymarmot/Paw-ObjCAFNetworkingCodeGenerator#3]
This generates code for AFNetworking 2.0. Should a new generator be created for the latest version or updated? In which case, users with AFNetworking 2.0 won't benefit from this. I might be able to submit a PR if I have some time.

**Comment by mittsh** [comment]
@revolter yes it's something we want to do for a while, but due to lack of time, it didn't happen so far...

**Comment by QIcareful** [comment]
@mittsh  Whether to give up the update?

**Comment by mittsh** [comment]
We haven't got time to work on this yet! I'll push that up the list :)

**Comment by tvlc** [comment]
@mittsh Are you going to update for AFNetworking 3.0?

**Comment by ichbinder** [comment]
@mittsh  It will be really nice to update for AFNetworking 3.0 :).
